Frontend/src/constants/pageUrls.js

```diff
@@ -2,5 +2,6 @@ export const pageUrls = {
     LOGIN: "/login",
     REGISTRATION: "/registration",
     CLIENTS_BRIEF_INFO: "/clients/brief-info",
-    CATEGORIES_VIEW: "/categories/view"
+    CATEGORIES_VIEW: "/categories/view",
+    VIEW_PROFILE_INFO: "/clients/profile-info"
```

Comment:
"/clients..." ? Are you sure? This is the route to view personal information, not client information